src/site/content/en/secure/fixing-mixed-content/index.md

```diff
@@ -0,0 +1,232 @@
+This means you will have to view every page of your site individually to find these errors.
+{% endAside %}
+
+### Finding mixed content in your site
+
+You can search for mixed content directly in your source code.
+Search for `http://` in your source and look for tags that include HTTP URL attributes.
+Note that having `http://` in the `href` attribute of anchor tags (`<a>`)
+is often not a mixed content issue, with some notable exceptions discussed later.
+
+If your site is published using a content management system
+it is possible that the links to insecure URLs have been inserted into the content when publishing pages.
+For example, images may have been included with a full URL rather than a relative path.
+You will need to find and fix these within the CMS content.
+
+### Fixing mixed content
+
+Once you've found where the mixed content is included in your site's source,
+follow these steps to fix it.
+
+First, find the resource and check to see if it is available securely by changing the `http://` to `https://`.
+If the Console message says that the resource has been automatically upgraded,
+then the resource should be available securely and you can change the URL to match.
+
+If the resource is not available via `https://`, you should consider one of the following options:
+
+* Include the resource from a different host, if one is available.
+* Download and host the content on your site directly, if you are legally allowed to do so.
+* Exclude the resource from your site altogether.
+
+Having fixed the problem,
+view the page where you found the error originally and verify that the error no longer appears.
+
+### Beware of non-standard tag usage
+
+Beware of non-standard tag usage on your site.
+For instance, anchor (`<a>`) tag URLs don't cause mixed content by themselves,
+as they cause the browser to navigate to a new page.
+This means they usually don't need to be fixed.
+However some image gallery scripts override the functionality of the `<a>`
+tag and load the HTTP resource specified by the `href` attribute into a lightbox display on the page,
+causing a mixed content problem.
+
+## Handle mixed content at scale
+
+The manual steps above work well for smaller websites;
+but for large websites or sites with many separate development teams,
+it can be tough to keep track of all the content being loaded.
+To help with this task, you can use content security policy
+to instruct the browser to notify you about mixed content and ensure that your pages never unexpectedly load insecure resources.
+
+### Content security policy
+
+[Content security policy](https://developers.google.com/web/fundamentals/security/csp/) (CSP)
+is a multi-purpose browser feature that you can use to manage mixed content at scale.
+The CSP reporting mechanism can be used to track the mixed content on your site;
+and the enforcement policy, to protect users by upgrading or blocking mixed content.
```

Comment:
This sounds a bit odd. Maybe 'The CSP reporting mechanism can be used to track mixed content on your site, and provide enforcement policies to protect users by upgrading or blocking mixed content.'